src/overlays/actors/ovl_En_Horse/z_en_horse.h

```diff
@@ -4,50 +4,192 @@
+    /* 0 */ ENHORSE_ANIM_IDLE,
+    /* 1 */ ENHORSE_ANIM_WHINNEY,
+    /* 2 */ ENHORSE_ANIM_STOPPING,
+    /* 3 */ ENHORSE_ANIM_REARING,
+    /* 4 */ ENHORSE_ANIM_WALK,
+    /* 5 */ ENHORSE_ANIM_TROT,
+    /* 6 */ ENHORSE_ANIM_GALLOP,
+    /* 7 */ ENHORSE_ANIM_LOW_JUMP,
+    /* 8 */ ENHORSE_ANIM_HIGH_JUMP,
+} EnHorseAnimationIndex;
+
+typedef enum {
+    /* 0 */ HORSE_EPONA,
+    /* 1 */ HORSE_HNI,
+} HorseType;
+
+typedef void (*EnHorsePostdrawFunc)(struct EnHorse*, GlobalContext*);
+
 typedef struct EnHorse {
-    /* 0x000 */ Actor actor;
-    /* 0x014C */ s32 unk_14C;
-    /* 0x0150 */ char unk_150[0x008];
-    /* 0x0150 */ s32 unk_158;
-    /* 0x0150 */ char unk_15C[0x094];
-    /* 0x01F0 */ u32 unk_1F0;
-    /* 0x01F4 */ char unk_1F4[0x01C];
-    /* 0x0210 */ s32 unk_210;
-    /* 0x0214 */ f32 unk_214;
-    /* 0x0218 */ char unk_218[0x040];
-    /* 0x0258 */ Vec3f unk_258;
-    /* 0x0264 */ char unk_264[0x124];
-    /* 0x0388 */ s32 unk_388;
-    /* 0x038C */ char unk_38C[0x14];
-    /* 0x03A0 */ s32 unk_3A0;
-    /* 0x03A4 */ char unk_3A4[0x58];
+    /* 0x0000 */ Actor actor;
+    /* 0x014C */ EnHorseAction action;
+    /* 0x0150 */ s32 noInputTimer;
+    /* 0x0154 */ s32 noInputTimerMax;
+    /* 0x0158 */ s32 type;
+    /* 0x015C */ s8 bankIndex;
+    /* 0x0160 */ PSkinAwb skin;
+    /* 0x01F0 */ u32 flags;
+    /* 0x01F4 */ Vec3f lastPos;
+    /* 0x0200 */ s16 lastYaw;
+    /* 0x0204 */ s32 curRaceWaypoint;
+    /* 0x0208 */ s32 boostSpeed;
+    /* 0x020C */ s32 playerControlled;
+    /* 0x0210 */ s32 animationIdx;
+    /* 0x0214 */ f32 curFrame;
+    /* 0x0218 */ s32 soundTimer;
+    /* 0x021C */ Vec3f unk_21C;
+    /* 0x0228 */ Vec3f unk_228;
+    /* 0x0234 */ s32 unk_234;
+    /* 0x0238 */ u8 numBoosts;
+    /* 0x023C */ s32 boostRegenTime;
+    /* 0x0240 */ s32 boostTimer;
+    /* 0x0244 */ EnHorsePostdrawFunc postDrawFunc;
+
+    // The y coordinate of the floor under the front and back feet
+    /* 0x0248 */ f32 yFront;
+    /* 0x024C */ f32 yBack;
+
+    /* 0x0250 */ s16 followTimer;
+    /* 0x0252 */ s16 unk_252;
+    /* 0x0254 */ EnHorseAction prevAction;
+    /* 0x0258 */ Vec3f riderPos;
+    /* 0x0264 */ Vec2f curStick;
+    /* 0x026C */ Vec2f lastStick;
+    /* 0x0274 */ f32 jumpStartY;
+    /* 0x0278 */ ColliderCylinder cyl1;
+    /* 0x02C4 */ ColliderCylinder cyl2;
+    /* 0x0310 */ ColliderJntSph jntSph;
+    /* 0x0330 */ ColliderJntSphElement jntSphList;
+    /* 0x0370 */ u32 playerDir;
+    /* 0x0374 */ s16 unk_374;
+    /* 0x0376 */ s16 angleToPlayer;
+    /* 0x0378 */ s16 followPlayerTurnSpeed;
+    /* 0x037A */ u8 blinkTimer;
+    /* 0x037C */ s16 waitTimer;
+    /* 0x037E */ s16 unk_37E;
+    /* 0x0380 */ s32 cutsceneAction;
+    /* 0x0384 */ u16 cutsceneFlags;
+    /* 0x0386 */ s16 unk_386; // pad
+    // struct {
+    /* 0x0388 */ s32 inRace;
+    /* 0x038C */ Actor* rider;
+    /* 0x0390 */ u32 unk_390;
+    /* 0x0394 */ u16 ingoRaceFlags;
+    /* 0x0398 */ f32 ingoHorseMaxSpeed;
+    // } race; //?
```

Comment:
the assert string confirms this is definitely real. guess we just need to decide if we want to do it as well, if it makes sense to. we should decide that now probably